File: EFCore Security Applications/EFCoreSecurityODataService/EFCoreSecurityODataService/Models/DemoTask.cs
using System;
using System.Collections.Generic;

namespace EFCoreSecurityODataService.Models {
    public class DemoTask {
        public int Id { get; set; }
        public string Description { get; set; }
        public string Note { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime DateCompleted { get; set; }
        public int PercentCompleted { get; set; }
        public List<ContactTask> ContactTasks { get; set; }
    }
}

File: EFCore Security Applications/EFCoreSecurityAndroidDemo/app/src/main/java/com/devexpress/efcoresecurity/efcoresecuritydemo/businessobjects/DemoTask.java
package com.devexpress.efcoresecurity.efcoresecuritydemo.businessobjects;

import java.security.Timestamp;
import java.util.ArrayList;
import java.util.Calendar;
import java.util.Collection;

/**
 * Created by neroslavskiy.a on 4/6/2016.
 */
public class DemoTask extends BaseSecurityEntity {
    public String Description;
    public String Note;
    public Calendar StartDate;
    public Calendar DateCompleted;
    public int PercentCompleted;
    public ArrayList<ContactTask> ContactTasks;
}